tests/requirements.yml

```diff
@@ -1,6 +1,3 @@
-- name: openmicroscopy.omero-common
-  src:  https://github.com/openmicroscopy/ansible-role-omero-common.git
-- name: openmicroscopy.omego
-  src:  https://github.com/openmicroscopy/ansible-role-omego.git
-- name: openmicroscopy.omero-web
-  src:  https://github.com/manics/ansible-role-omero-web.git
+- role: openmicroscopy.omero-common
```

Comment:
Minor code-style issue: I've been using `- src: openmicroscopy.role` instead of `role: ...`